lib/open_project/translations/helpers/tmp_directory.rb

```diff
@@ -0,0 +1,14 @@
+module TmpDirectory
+  def create_tmp_directory(delete: nil)
```

Comment:
@jonasheinrich why do you use a default value of `nil ` here when you are using it like a boolean? `false` would be better.